src/core/utils/compatibility.js

```diff
@@ -127,6 +128,13 @@ export function isCompatibleWithUserAgent({
     return { compatible: false, reason: INCOMPATIBLE_NOT_FIREFOX };
   }
 
+  // Fenix does not support add-ons (yet?).
+  // See: https://github.com/mozilla-mobile/fenix/issues/1134
+  // See also: https://github.com/mozilla/addons-frontend/issues/7963
+  if (os.name === 'Android' && mozCompare(browser.version, '69.0') >= 0) {
```

Comment:
I don't see the greater than 69 part covered in a test. Can you add one with a Fenix 70 UA or something like that?

Reply:
ah yes, I did a manual check but a test case will be more future-proof, thanks